diff --git a/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js b/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js
index a533efb..560cb87 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js
@@ -29,10 +29,10 @@ async function main() {
     }
 
     const packageConf = JSON.parse(await readFile("package.json"));
-    const widgetVersion = packageConf?.version;
+    const widgetVersion = packageConf.version;
 
     // Downloading test project
-    if (!process.argv.includes("--no-update-testProject") && packageConf?.testProject?.githubUrl && packageConf?.testProject?.branchName) {
+    if (!process.argv.includes("--no-update-testProject") && packageConf.testProject.githubUrl && packageConf.testProject.branchName) {
         await unzipTestProject();
     } else {
         const projectMpr = ls(`tests/testProject/*.mpr`).length;
@@ -159,7 +159,7 @@ async function getMendixVersion() {
 
         const mendixVersionsJson = await mendixVersions.json();
 
-        mendixVersion = mendixVersionsJson["8"];
+        mendixVersion = mendixVersionsJson.latest;
     } catch (e) {
         throw new Error("Couldn't reach github.com. Make sure you are connected to internet.");
     }
diff --git a/node_modules/@mendix/pluggable-widgets-tools/scripts/mxbuild.Dockerfile b/node_modules/@mendix/pluggable-widgets-tools/scripts/mxbuild.Dockerfile
index da7f229..7885937 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/scripts/mxbuild.Dockerfile
+++ b/node_modules/@mendix/pluggable-widgets-tools/scripts/mxbuild.Dockerfile
@@ -1,4 +1,4 @@
-FROM mono:5.20.1
+FROM mono:6.12
 ARG MENDIX_VERSION
 
 RUN \
