diff --git a/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js b/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js
index bcdca41..750d798 100644
--- a/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js
+++ b/node_modules/@mendix/pluggable-widgets-tools/scripts/e2e.js
@@ -30,7 +30,7 @@ async function main() {
     }
 
     const packageConf = JSON.parse(await readFile("package.json"));
-    const widgetVersion = packageConf?.version;
+    const widgetVersion = packageConf.version;
     const widgetMpk = ls(`dist/${widgetVersion}/*.mpk`).length;
 
     if (!widgetMpk) {
